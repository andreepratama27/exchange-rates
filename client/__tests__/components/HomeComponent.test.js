import React from "react";
import { shallow } from "enzyme";
import "./../jestsetup";

describe("Make sure test running", () => {
  test("render a label", () => {
    const wrapper = shallow(<label>Andre</label>);
    expect(wrapper).toMatchSnapshot();
  });
});
