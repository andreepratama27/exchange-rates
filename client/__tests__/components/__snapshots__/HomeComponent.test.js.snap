// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make sure test running render a label 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <label>
    Andre
  </label>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Andre",
    },
    "ref": null,
    "rendered": "Andre",
    "type": "label",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Andre",
      },
      "ref": null,
      "rendered": "Andre",
      "type": "label",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;

exports[`render a label 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <label>
    Andre
  </label>,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "Andre",
    },
    "ref": null,
    "rendered": "Andre",
    "type": "label",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Andre",
      },
      "ref": null,
      "rendered": "Andre",
      "type": "label",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
      },
    },
  },
}
`;
